file_store/src/cli/bucket.rs

```diff
@@ -217,7 +217,6 @@ fn locate(
         FileType::LoraValidPoc => {
             LoraValidPoc::decode(buf).and_then(|event| event.to_value_if(gateway))
         }
-        FileType::EntropyReport => Ok(None),
```

Comment:
do we no longer need to report these for validation?

Reply:
Fair enough.. I was trying to be explicit, but it's superfluous